winforms/src/toga_winforms/widgets/slider.py

```diff
@@ -10,69 +11,32 @@
 
 BOTTOM_RIGHT_TICK_STYLE = WinForms.TickStyle.BottomRight
 
-DEFAULT_NUMBER_OF_TICKS = 10000
-
-
-class Slider(Widget):
-    """Implementation details:
-
-    The slider widget is using .Net "TrackBar" class. Since TrackBar can only be
-    discrete (ie. have integer values), we implement our slider as a TrackBar
-    between 0 and tick_count. In order to have value between the desired minimum
-    and maximum, we trnaslate the value linearly to the desired interval.
-
-    When tick_count is not defined, we use 100 as the default number of ticks since
-    it is big enough to make the TrackBar feel continuous.
```

Comment:
The details behind this comment aren't entirely correct any more, but the sprit of them - explaining the general implementation approach - would be worth preserving. A quick reminder that the Winforms implementation is integer based, so it leans on the IntSliderImpl with 10k discrete positions  etc isn't necessarily a detail that will stand out unless you're familiar with the code, so pointing it out explicitly in a comment could be helpful. 

(The same is true for the other implementations that use the Int/Continuous slider impls).